src/main/java/com/dev/cinema/config/AppConfig.java

```diff
@@ -0,0 +1,63 @@
+package com.dev.cinema.config;
+
+import com.dev.cinema.model.CinemaHall;
+import com.dev.cinema.model.Movie;
+import com.dev.cinema.model.MovieSession;
+import com.dev.cinema.model.Orders;
+import com.dev.cinema.model.ShoppingCart;
+import com.dev.cinema.model.Ticket;
+import com.dev.cinema.model.User;
+
+import java.util.Properties;
+import javax.sql.DataSource;
+
+import org.apache.commons.dbcp2.BasicDataSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.orm.hibernate5.LocalSessionFactoryBean;
+
+@Configuration
+@PropertySource("classpath:db.properties")
+@ComponentScan(basePackages = {
+        "com.dev.cinema.dao",
+        "com.dev.cinema.service",
+        "com.dev.cinema.controllers"
+})
+public class AppConfig {
+
+    @Autowired
+    private Environment environment;
+
+    @Bean
+    public DataSource getDataSource() {
+        BasicDataSource basicDataSource = new BasicDataSource();
+        basicDataSource.setDriverClassName(environment.getProperty("db.driver"));
+        basicDataSource.setUrl(environment.getProperty("db.url"));
+        basicDataSource.setUsername(environment.getProperty("db.username"));
+        basicDataSource.setPassword(environment.getProperty("db.password"));
+        return basicDataSource;
+    }
+
+    @Bean
+    public LocalSessionFactoryBean sessionFactoryBean() {
```

Comment:
```suggestion
    public LocalSessionFactoryBean getSessionFactoryBean() {
```
назва методу - дієслова або фраза на його основі (get, put, createNewFile)